lib/http2/stream.c

```diff
@@ -271,12 +272,40 @@ static int send_headers(h2o_http2_conn_t *conn, h2o_http2_stream_t *stream)
         if (is_blocking_asset(&stream->req))
             h2o_http2_scheduler_rebind(&stream->_scheduler, &conn->scheduler, 257, 0);
     } else {
-        /* raise the priority of asset files that block rendering to highest if the user-agent is _not_ using dependency-based
-         * prioritization (e.g. that of Firefox)
-         */
-        if (conn->num_streams.priority.open == 0 && stream->req.hostconf->http2.reprioritize_blocking_assets &&
-            h2o_http2_scheduler_get_parent(&stream->_scheduler) == &conn->scheduler && is_blocking_asset(&stream->req))
+        /* Handle absolute priority header */
+        ssize_t absprio_cursor = h2o_find_header_by_str(&stream->req.res.headers, H2O_STRLIT("priority"), -1);
```

Comment:
it might be worth adding `priority` to the tokens: https://github.com/h2o/h2o/blob/master/misc/tokens.pl

Reply:
Done.